notebooks/file_calibration.ipynb

```diff
@@ -61,6 +61,7 @@
     "from hera_notebook_templates.data import DATA_PATH as HNBT_DATA\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #33.        cross_wgt = datacontainer.DataContainer(cross_wgt)
</pre><p>I'm not sure that these need to be datacontainers</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_notebook_templates/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Reply:
Yeah, that's probably right. I've removed these